src/main/java/com/minecolonies/coremod/entity/EntityCitizen.java

```diff
@@ -431,11 +437,66 @@ public void onJobChanged(@Nullable final AbstractJob job)
         }
     }
 
+    /**
+     * Get the latest status of the citizen.
+     * @return a ITextComponent with the length 4 describing it.
+     */
+    public ITextComponent[] getLatestStatus()
+    {
+        return latestStatus.clone();
+    }
+
+    //todo we have add to latest status and setLatest status
```

Comment:
![INFO](https://sonarsource.github.io/sonar-github/severity-info.png 'Severity: INFO') Complete the task associated to this TODO comment. [![rule](https://sonarsource.github.io/sonar-github/rule.png)](https://sonar.minecolonies.com/coding_rules#rule_key=squid%3AS1135)